shop4All-backend/src/main/java/com/example/shop4All_backend/Controller/UserController.java

```diff
@@ -19,17 +19,32 @@ public class UserController {
     @Autowired
     private UserService userService;
 
+
+
     //initialize a demo admin
     @PostConstruct
     public void initRolesAndUsers(){
         userService.initRolesandUser();
     }
 
-    // Handle POST requests to /registerNewUser to register a new user
+    // Handle POST requests to /registerNewSeller to register a new seller
     @PostMapping({"/registerNewSeller"})
-    public ResponseEntity<?> registerNewSeller(@RequestBody User user) {
+    public ResponseEntity<?> registerNewSeller(@RequestBody Seller seller) {
```

Comment:
If you know the type of the returned ResponseEntity, please add the Object type between "<>". Do the same for every ResponseEntity<?>.